File: ass.js
function greetings(){
  // Ask the user for their name
  let name = prompt("What is your name?");

  // Ask the user for their current age
  let age = parseInt(prompt("How old are you?"));

  // Get the current year
  let currentYear = new Date().getFullYear();

  // Calculate birth year
  let birthYear = currentYear - age;

  // Calculate age in five years
  let ageInFiveYears = age + 5;

  // Display the result
  alert(
    `Welcome to the practical world, ${name}!\n` +
    `Your current age is ${age}.\n` +
    `You were born in ${birthYear}.\n` +
    `In five years, you will be ${ageInFiveYears} years old.`
  );

}

console.log(greetings())

File: ass.py
# Ask user for their name
name = input("What is your name? ")

# Ask user for their current age
age = input("How old are you? ")

# Get the current year
from datetime import date
current_year = date.today().year

# Calculate birth year and age in five years
birth_year = current_year - age
age_in_five_years = age + 5

# Display the results
print(f"\nWelcome to the practical world, {name}!")
print(f"Your current age is {age}.")
print(f"You were born in {birth_year}.")
print(f"In five years, you will be {age_in_five_years} years old.")


